crates/hdi/Cargo.toml

```diff
@@ -25,7 +25,7 @@ hdk_derive = { version = "0.0.40", path = "../hdk_derive" }
 holo_hash = { version = "0.0.30", path = "../holo_hash" }
 holochain_wasmer_guest = "=0.0.80"
 # it's important that we depend on holochain_integrity_types with no default
-# features, both here AND in holochain_deterministic_integrity_derive, to reduce code bloat
+# features, both here AND in hdk_derive, to reduce code bloat
```

Comment:
i'm not sure whether this is accurate but the original _holochain_deterministic_integrity_derive_ that was in place here is a non-existing crate

Reply:
the only `_derive` crate there is

@freesig Is that the correct replacement?